modules/nw-multus-ipam-object.adoc

```diff
@@ -310,4 +310,47 @@ pod/whereabouts-reconciler-p4sxw 1/1 Running 0 6s
 pod/whereabouts-reconciler-rvfdv 1/1 Running 0 6s
 pod/whereabouts-reconciler-svzw9 1/1 Running 0 6s
 daemonset.apps/whereabouts-reconciler 6 6 6 6 6 kubernetes.io/os=linux 6s
-----
\ No newline at end of file
+----
+
+[id="nw-multus-creating-whereabouts-dualstack-config_{context}"]
+== Creating a DualStack config
+
+DualStack can be configured with `ipRanges`.
+
+.Procedure
+
+. Set type to whereabouts
+. Use `ipRanges` to allocate IP addresses.
+
+For example, defining the range as `192.168.10.1/24` excludes the first network address and the last broadcast address.
+
+
+[source,yaml]
+----
+cniVersion: operator.openshift.io/v1
+kind: Network
+metadata:
+  name: cluster
+spec:
+  additionalNetworks:
+  - name: whereabouts-shim
+    namespace: default
+    rawCNIConfig: |-
+      {
+       "name": "whereabouts-shim",
+       "cniVersion": "0.3.1,
+       "type": "bridge",
+       "ipam": {
+         "type": "whereabouts",
+         "ipRanges": [{
+           "range": "192.168.10.1/24"
+         }, {
+           "range": "abcd: :1/64"
```

Comment:
let's use a documentation range for IPv6, from: https://www.rfc-editor.org/rfc/rfc3849

Such as: 

```
2001:db8::/64
```